demos/motion_tracking/main_motion_tracking.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+from threading import Thread
+from functools import partial
```

Comment:
Can we import these as
```python
import threading
import functools
import multiprocessing
```
to clarify later in the code where the imported objects come from?

I would prefer something similar for the `bokeh` imports but its deep module hierarchy does not work well with that.

Reply:
Done